providers/fab/tests/provider_tests/fab/www/views/test_views_acl.py

```diff
@@ -85,6 +94,25 @@
 }
 
 
+def _get_appbuilder_pk_string(model_view_cls, instance) -> str:
```

Comment:
I moved a number of tests from "airflow/www" to "fab" -> because those tests are testing permission models for FAB rather than views themselves and they are using some "fab-specific" api connection utils (create user and such) wich should no longer be used from "airlfow" core once the "fab" provicder tests code is truly isolated from airflow core. There was already a number of tests like that moved before but this changed allowed to find and detect remaining ones, and as a side-effect airflow tests should no longer (except some structural tests that auto-detec provider example dags and import it) import anything from FAB. cc: @vincbeck